examples/launch_height_mosgpfifo.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
exactly the same as `examples/launch_height_moasha.py` but with my new `MOScalarFIFOScheduler`